DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -298,5 +300,23 @@ int customisationId
                 new { customisationId, centreId, categoryId }
             ).FirstOrDefault();
         }
+
+        public IEnumerable<Course> GetCoursesAtCentreForCategoryId(int centreId, int categoryId)
+        {
+            return connection.Query<Course>(
+                @"SELECT
+                        c.CustomisationID,
+                        c.CentreID,
+                        c.ApplicationID,

```

Comment:
Do we need the application id?

Reply:
Not until I commonised CourseDetails too